docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,34 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
```

Comment:
Please consider reducing one `#` on each newly added heading.
```suggestion
## Migration based hotplug
```

Reply:
I actually think this makes sense.

Reply:
Done